### PR TITLE
[SECURITY-2186] follow-up documentation

### PR DESCRIPTION
**Pull request by Pldi23**
[SECURITY-2186](https://issues.jenkins.io/browse/SECURITY-2186) was already pushed to master but without properly written description, I think it would be useful to update readme

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira: fix [commit](https://github.com/jenkinsci/support-core-plugin/commit/c6d20da4f372f03bd3e4844f0df2f109df68a63c) link
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by rsandell** [comment]
@jenkinsci/company-cloudbees-developers FYI

**Review by Dohbedoh — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -198,3 +198,34 @@ unclassified:
...
+nodes/master/system.properties for controllers and agents
+
+nodes/master/environment.txt
+
+nodes/slave/name/proc/self/environ and nodes/slave/name/proc/self/cmdline for Linux based instances
+
+nodes/slave/name/config.xml for each agent
+
+launch log files per each agent located in nodes/slave/name/launchLogs
+````
```
Comment (Dohbedoh) [inline README.md:219]:
```suggestion
* `nodes.md`
* `about.md`
* `nodes/master/system.properties` for controllers and agents
* `nodes/master/environment.txt`
* `nodes/slave/name/proc/self/environ` and `nodes/slave/name/proc/self/cmdline` for Linux based instances
* `nodes/slave/name/config.xml` for each agent
* launch log files per each agent located in `nodes/slave/name/launchLogs`
```

File: README.md [hunk 2]
```diff
@@ -198,3 +198,34 @@ unclassified:
...
+
+To support this feature, a text file named security-stop-words.txt has been added to the JENKINS_HOME/support folder. It contains security stop words that are used to detect passwords or secrets. When the bundle is generated, the word "REDACTED" replaces any values associated with these stop words.
```
Comment (Dohbedoh) [inline README.md:221]:
```suggestion
To support this feature, a text file named `security-stop-words.txt` has been added to the `$JENKINS_HOME/support` folder. It contains security stop words that are used to detect passwords or secrets. When the bundle is generated, the word "REDACTED" replaces any values associated with these stop words.
```

**Review by Dohbedoh — APPROVED** [review]
LGTM with syntax suggestions

**Comment by Dohbedoh** [comment]
Thanks @Pldi23 !

**Pull request merged**